packages/sdk/src/types/runtime/sound.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SetSoundStateOptions } from '../../sound';
+
+export class SoundInstance {
```

Comment:
>SoundInstance [](start = 13, length = 13)

I have a hunch SoundInstance will need a reference to either the AssetManager or the Context so that it can send the `set-sound-state` message (my recommended replacement for the 'play-sound' message).

Reply:
yep